drivers/display/display_st7789v.c

```diff
@@ -22,7 +22,6 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(display_st7789v);
 
-#define ST7789V_CS_PIN		DT_INST_SPI_DEV_CS_GPIOS_PIN(0)
```

Comment:
There's another PR cleaning up ST7789 (including other changes): https://github.com/zephyrproject-rtos/zephyr/pull/36717

Reply:
Happy to drop the ST7789 commit from this PR.